docs/_advanced/custom_routes.md

```diff
@@ -6,9 +6,43 @@ order: 10
 ---
 
 <div class="section-content">
+As of `v3.7.0`, custom HTTP routes can be easily added by passing in a collection of routes as `customRoutes` when initializing `App`. 
 
-Adding custom HTTP routes is quite straightforward when using Bolt's built-in `ExpressReceiver`. Since `v2.1.0`, `ExpressReceiver` added a `router` property, which exposes the Express [Router](http://expressjs.com/en/4x/api.html#router) on which more routes can be added.
+Each `CustomRoute` object must contain three properties: `path`, `method`, and `handler`. `method`, which corresponds to the HTTP verb, can be either a string or an array of strings.
+</div>
+
+```javascript
+const { App } = require('@slack/bolt');
+
+// Create the Bolt App, using the receiver
```

Comment:
```suggestion
// Initialize Bolt app, using the default HTTPReceiver
```

Reply:
Oops, this probably shouldn't even be there. This was carry over from the existing example.